openassessment/assessment/api/peer.py

```diff
@@ -483,7 +483,7 @@ def has_finished_required_evaluating(submission_uuid, required_assessments):
     return done, peers_graded
 
 
-def get_assessments(submission_uuid, scored_only=True, limit=None):
+def get_assessments(submission_uuid, scored_only=False, limit=None):
```

Comment:
Similar to my comment below, is get_assessments ever used with scored_only=True outside of tests? If not, should we just delete the argument from the signature?